python/tk_multi_publish2/processing/plugin.py

```diff
@@ -283,18 +283,9 @@ def run_accept(self, item):
         :param item: Item to analyze
         :returns: dictionary with boolean keys accepted/visible/enabled/checked
         """
-        try:
+
+        with self._handle_plugin_error(None, "Error running accpet: %s"):
             return self._hook_instance.accept(self.settings, item)
-        except Exception, e:
-            error_msg = traceback.format_exc()
-            self._logger.error(
-                "Error running accept for %s" % self,
-                extra=_get_error_extra_info(error_msg)
-            )
-            return {"accepted": False}
```

Comment:
You've changed the behaviour here. The method used to return {"accepted": False} when there was an error in the plugin, not it raises an exception.

Reply:
reverted. thanks!